src/components/CheckIn.jsx

```diff
@@ -9,6 +11,23 @@ const mockuser = {
   email: "big.chungus@email.com",
   pfp: LOGO,
 };
+
+const Timer = () => {
+  const date = new Date();
```

Comment:
use this value directly within the useState. You can simplify the code to just use the useState and eliminate the need for the second variable